dashboard/lib/policies/lti.rb

```diff
@@ -101,6 +112,13 @@ def self.issuer_name(issuer)
     I18n.t(:lti_v1, scope: [:section, :type])
   end
 
+  # Returns the LTI user id for a particular code.org user and LTI integration
+  def self.lti_user_id(user, lti_integration)
+    lti_user_identities = user.lti_user_identities
+
+    lti_user_identities.find {|identity|  identity.lti_integration_id == lti_integration.id}.try(:subject)
```

Comment:
Instead of `find` does `find_by` work, and keep it a little cleaner? I'm certainly no ActiveRecord expert so I'm not sure if it works or not:
```suggestion
    user.lti_user_identities.find_by(lti_integration_id: lti_integration.id)&.subject
```